store/tikv/coprocessor.go

```diff
@@ -1345,13 +1350,9 @@ func newRateLimitAction(totalTokenNumber uint) *rateLimitAction {
 
 // Action implements ActionOnExceed.Action
 func (e *rateLimitAction) Action(t *memory.Tracker) {
-	failpoint.Inject("testRateLimitActionDisable", func(val failpoint.Value) {
-		if val.(bool) {
-			e.setEnabled(false)
-		}
-	})
-
 	if !e.isEnabled() {
+		logutil.BgLogger().Info("memory exceed quota, rateLimitAction delegate to fallback action",
```

Comment:
The log is necessary? If there 5 rateLimitActions in the action chain, it will print 4 fallback log and 1 trigger successfully log.....  It maybe print too many logs?